.github/workflows/redis-recovery.yml

```diff
@@ -0,0 +1,37 @@
+name: Redis recovery in Openshift
+run-name: Redis recovery in Openshift from ${{ inputs.gitRef }} to ${{ inputs.environment }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
+        description: "Build Ref"
+        required: true
+        default: ""
+
+jobs:
+  recoverRedis:
+    name: Redis recovery in Openshift
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Print env
+        run: |
+          echo NAMESPACE: $NAMESPACE
```

Comment:
GitHub will not print a value that came from a secret so there is no point in printing it, IMO.